save-cloud-charts/save-cloud/Chart.lock

```diff
@@ -1,15 +1,15 @@
 dependencies:
 - name: grafana
   repository: https://grafana.github.io/helm-charts
-  version: 6.31.1
+  version: 6.32.10
```

Comment:
does renovate able to upgrade it? I don't remember PRs for it

Reply:
I was hoping that's what this setting is for
https://github.com/saveourtool/save-cloud/blob/0cd164a405d9440d4c9003d5cb21af1a16126f07/renovate.json#L45-L47

But apparently, it needs something else, like global toggle for lock files maintenance. The problem is that we also have a `yarn.lock`, which shouldn't be updated automatically, so this needs some more experiments.